frontend/src/components/annotation/annotation_ui_factory.vue

```diff
@@ -1337,6 +1343,7 @@ export default Vue.extend({
                 )
             ) return
             this.set_save_loading(true, frame_number);
+            instance_list = to_serializable_instance_list(instance_list)
```

Comment:
context:
circular dependency with new annotation classes
this removes that (object dependencies related to front end)

e.g errors when copying or saving